packages/contracts-bedrock/src/L1/SimpleStorage.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Initializable } from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+// Libraries
+import { Storage } from "src/libraries/Storage.sol";
+
+contract SimpleStorage is Initializable {
+    // Storage slots used to manage upgrades
+    bytes32 internal constant UPGRADED_1_3_0 = keccak256("UPGRADED_1_3_0");
```

Comment:
Thoughts on a single storage slot that has the version rather than many with a bool? I think ops will be easier with a single slot

Reply:
this sounds like a good proposal, I will play around with that